AutomationFramework/src/test/java/steam/pages/MainPage.java

```diff
@@ -0,0 +1,44 @@
+package steam.pages;
+
+import framework.driver.Browser;
+import framework.page.BasePage;
+import framework.pageElements.Button;
+import framework.pageElements.Label;
+import org.openqa.selenium.By;
+
+import java.time.Duration;
+
+
+public class MainPage extends BasePage {
+    protected final Label mainPageUniqueLabel = new Label(By.xpath("//div[contains(@class, 'capsule main_capsule')]"));
+    private final Button categoriesButton = new Button(By.xpath("//a[contains(text(), 'Categories')]"));
+    private final Button newAndNoteworthyButton = new Button(By.xpath("//a[contains(text(), 'New & Noteworthy')]"));
+    private final Button specialOffersButton = new Button(By.xpath("//a[contains(text(), 'Special Offers')]"));
+    private final Label newAndTrendingLabel = new Label(By.xpath("//div[@class=\"home_ctn tab_container\"]"));
+    private final Button goToMainPageButton = new Button(By.xpath("//span[@id=\"logo_holder\"]"));
+
+    public MainPage() {
+        uniqueElement = mainPageUniqueLabel;
+    }
+
+    public void navigateToCategories() {
```

Comment:
rename to `hoverOnCategories`, because navigate used when you telling your browser to go to some url